### PR TITLE
adding tokens to cookie

### PR DESCRIPTION
**Pull request by monoskribt**

### CODE REVIEW

File: src/main/java/com/spotifyapi/dto/TokensDTO.java [hunk 1]
```diff
@@ -11,5 +11,6 @@ public class TokensDTO {
 
     private String accessToken;
     private String refreshToken;
+    private String accessCode;
```
Comment (syevtushok) [inline src/main/java/com/spotifyapi/dto/TokensDTO.java]: Do you use accessCode somewhere? Does it change?

File: src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@AllArgsConstructor
+public class CookieServiceImpl implements CookieService {
+
+    @Override
+    public void setCookie(HttpServletResponse response, TokensDTO tokens) {
+        Cookie cookieAccessToken = new Cookie("access_token", tokens.getAccessToken());
+        Cookie cookieRefreshToken = new Cookie("refresh_token", tokens.getRefreshToken());
+
+        cookieAccessToken.setMaxAge(24 * 60 * 60);
+        cookieRefreshToken.setMaxAge(24 * 60 * 60);
```
Comment (syevtushok) [inline src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java]: how long is the access token from Spotify valid? You should set the same time

File: src/main/java/com/spotifyapi/dto/TokensDTO.java (hunk 1 above)
Comment (monoskribt) [reply src/main/java/com/spotifyapi/dto/TokensDTO.java]: Yes, i used it for store access code in DB

File: src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java (hunk 2 above)
Comment (monoskribt) [reply src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java]: Changed access token on 1 day

File: src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+public class CookieServiceImpl implements CookieService {
+
+    private final SpotifyAuth spotifyAuth;
+
+    @Override
+    public void setCookie(HttpServletResponse response, TokensDTO tokens) {
+        Cookie cookieAccessToken = new Cookie("access_token", tokens.getAccessToken());
+        Cookie cookieRefreshToken = new Cookie("refresh_token", tokens.getRefreshToken());
+
+        cookieAccessToken.setMaxAge(60 * 60);
```
Comment (syevtushok) [inline src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java:27]: move it to constant and give it a name

**Review by syevtushok — APPROVED** [review]

File: src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java (hunk 3 above)
Comment (monoskribt) [reply src/main/java/com/spotifyapi/service/impl/CookieServiceImpl.java:27]: Done

**Pull request merged**